src/stim/search/sat/wcnf.test.cc

```diff
@@ -0,0 +1,55 @@
+#include "stim/search/sat/wcnf.h"
+#include "gtest/gtest.h"
+
+using namespace stim;
+
+TEST(shortest_undetectable_logical_error_wcnf, no_error) {
+    // No error.
+    ASSERT_THROW(
```

Comment:
Is this the right behavior for the empty case? Shouldn't it return a file that embeds an unsatisfiable problem instead of failing, in this case?

Failing fast is great. I like failing fast. But I also like my code to work on the empty string when its semantics are compatible with the method. I'd be quite cross if `''.startswith('')` raised an exception that I was probably doing the wrong thing, instead of returning true.

Reply:
We can certainly return an unsatisfiable wcnf in these cases.